Fixes Issue #11

ratracegrad opened the pull request.

anshumanv commented:
Hi @ratracegrad 

Thanks for picking this issue! Appreciate it! ✨ 

For the homepage I was thinking of having a grid like pintrest. WDYT?
![image](https://user-images.githubusercontent.com/21009455/67900476-8829a500-fb8a-11e9-871d-65c6ce71e206.png)

I think this vue package may be relevant -  https://github.com/WouterFlorijn/vue-stack-grid

Thanks!

ratracegrad commented:
I have added vue-stack-grid to give the homepage a look like Pinterest. Restyled to use the layout for vue-stack-grid. Added border-radius to all images to look like images in your picture above.  Check it out now. I think it has exactly what you wanted. Thanks for letting me submit to your repo for hacktoberfest.

anshumanv commented:
This looks superb @ratracegrad! Thank You! 🙏 

> Check it out now. I think it has exactly what you wanted. 

Looks perfect!

There are some merge conflicts though, can you please resolve them as well so we can merge this? Also, some unrelated commits are present in this PR from your last PR.

Thanks!
